src/main/java/com/sprint/mission/discodeit/entity/Channel.java

```diff
@@ -1,51 +1,38 @@
 package com.sprint.mission.discodeit.entity;
 
-import java.util.ArrayList;
-import java.util.List;
+import lombok.Getter;
+
+import java.io.Serializable;
+import java.time.Instant;
 import java.util.UUID;
 
-public class Channel {
+@Getter
+ public class Channel implements Serializable {
+
+    private static final long serialVersionUID = 1L;
     private final UUID id;
     private String channelName;
-    private final List<Message> messages;
-    private final long createdAt;
-    private long updatedAt;
+    private String description;
+    private final Instant createdAt;
+    private Instant updatedAt;
 
-    public Channel(String channelName) {
+    public Channel(String channelName, String description) {
         this.id = UUID.randomUUID();
         this.channelName = channelName;
-        this.messages = new ArrayList<>();
-        this.createdAt = System.currentTimeMillis();
-        this.updatedAt = System.currentTimeMillis();
-    }
-
-    public UUID getId() {
-        return id;
-    }
-
-    public String getChannelName() {
-        return channelName;
+        this.description = description;
+        this.createdAt = Instant.now();
+        this.updatedAt = Instant.now();
```

Comment:
짧은 시간이지만 createdAt과 updatedAt 시간이 차이날 수도 있을 것 같아요.  
```suggestion
        this.createdAt = Instant.now();
        this.updatedAt = this.createdAt;
```